tools/autograd/gen_autograd_functions.py

```diff
@@ -346,6 +346,10 @@
 # TODO: This is probably not exhaustive, but it's a start
 UNTRACEABLE_FUNCTIONS = VIEW_FUNCTIONS
 
+_HASH_LINK_TREE = "#link-tree/"
```

Comment:
instead of having something Buck specific here, how about just getting the relative path from above the tools directory?

We can identify the tools directory with `pathlib.Path(__file__).parent.parent`